src/locales/en.json

```diff
@@ -97,8 +97,9 @@
     "duration_header": "Duration",
     "duration_question": "How long is the activity, in minutes? (For a repeated activity: minutes per week?)",
     "intimate_risk_warning": "We have applied a minimum Activity Risk for kissing due to the risk involved in exchanging fluids.",
-    "person_risk_profile_question": "What is their risk profile?",
-    "person_risk_profile_header": "Risk Profile",
+    "behavior_question": "How did they spend their last 10 days?",
+    "behavior_header": "Behavior",
+    "person_risk_note": "Each nearby person has a person risk of <2>{{ personRisk }} $t(calculator.pointsdisplay.microCOVIDs)</2> this week. Person risk (chance they currently have COVID) is calculated from their location, behavior, vaccination status, and infection status.",
```

Comment:
Is there a way to say this without saying "person risk"? It feels like there might be a way to un-jargon-ify it (since most users aren't going to ever read the white paper or look at the detailed calculation breakdown). 

I wonder about:
* "Each person nearby person has a score of 200 microCOVIDs over the past week" (I know, we also don't use "score" anywhere else in our documentation, so... no ideal)
* Or we could at least put it in quotes or italics. "Each nearby person has a *Person Risk* of 200 microCOVIDs"

Reply:
maybe "infection risk of 200 microCOVIDs" or is that not exactly what is meant here?